runtime/src/accounts_db.rs

```diff
@@ -1565,20 +1565,21 @@ impl AccountsDb {
         let total_keys_count = pubkeys.len();
         let mut accounts_scan = Measure::start("accounts_scan");
         // parallel scan the index.
-        let (mut purges, purges_in_root) = {
+        let (mut purges_zero_lamports, purges_in_root, purges_unrooted) = {
             self.thread_pool_clean.install(|| {
                 pubkeys
                     .par_chunks(4096)
                     .map(|pubkeys: &[Pubkey]| {
+                        let mut purges_zero_lamports = HashMap::new();
                         let mut purges_in_root = Vec::new();
```

Comment:
nits: another rename `purges_rooted` to be paired with `purges_unrooted`?

Reply:
Done!